tests/skyserve/llm/service.yaml

```diff
@@ -0,0 +1,39 @@
+resources:
```

Comment:
Just wondering why we are having another yaml here, but not using the one in examples?

Reply:
This test uses a small 3b model and T4 accelerator to reduce cost